Samples/Legacy/build.gradle

```diff
@@ -58,8 +50,7 @@ android {
         jvmTarget = "1.8"
     }
     composeOptions {
-        kotlinCompilerExtensionVersion versions.compose.core
-        kotlinCompilerVersion '1.7.21'
+        kotlinCompilerExtensionVersion "${versions.compose.compilerExt}"
```

Comment:
I'd prefer if we uncoupled the Sample from the shared versions. Perhaps bring back the `versions` block and set `compileExt` to match whatever is in the root build.gradle?